webpack.config.js

```diff
@@ -0,0 +1,104 @@
+const path = require("path");
+
+const webpack = require("webpack");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+
+const config = {
+    performance: {hints: false},
+    entry: {
+        i18n: ["./js/shared/i18n.js"],
+        output_pjs_deps: [
+            "./external/processing-js/processing.js",
+            "./external/jshint/jshint.js",
+        ],
```

Comment:
This is not very webpack-y but it does what we need.

Reply:
interesting - @kevinbarabash which part is not this not very `webpack-y`?

Reply:
I think the fact that I'm making a separate bundle for the dependencies, and the dependencies bundle exposes its stuff as globals. The output file makes that more clear, as you can see I include two bundles:
https://github.com/Khan/live-editor/pull/686/files#diff-315326e54ee98fc7fd456045df327d7d